brainstormforce/astra
# AST-614 - Added transparent header notices for above, primary, below header layout sections

### Description
- Added transparent header is enabled and some settings would work from their
- Accordingly added notice in header sections

### Screenshots
- Above header - https://share.bsf.io/NQuxR2P2
- Primary header - https://share.bsf.io/nOu9zAZZ
- Below header - https://share.bsf.io/Z4u1lQ0K

### Types of changes
- Non-breaking change 

### How has this been tested?
- Enable transparent header
- Check in header layouts then (primary/above/below)

### Checklist:
